x-pack/plugins/triggers_actions_ui/public/index.ts

```diff
@@ -80,6 +80,7 @@ export {
   MustacheTextFieldWrapper,
   SimpleConnectorForm,
   TextAreaWithMessageVariables,
+  TextAreaWithAutocomplete,
```

Comment:
You still need to delete `TextAreaWithAutocomplete` and then the bundle should go down.